build/ci/golangci.yml

```diff
@@ -44,9 +44,12 @@ linters-settings:
 
 # https://golangci-lint.run/usage/linters
 linters:
+  disable-all: true
```

Comment:
Predtym tu bolo `enabled` aj `disabled` (niektore linters su enabled by default), pridal som tu `disable-all`, a vymenoval vsetky linters v `enabled` (a oznacil komentarom vypnute), ... takto je to jednoznacne.